views/surveys/view_survey_results.php

```diff
@@ -0,0 +1,259 @@
+<?php
+global $params, $route, $view;
+include ('../models/Notification.php');
+$pages = ['view','edit','create','delete','restore'];
+# Update page title to reflect route
+if (!empty($params) && in_array($params[0], $pages)) {
+	$newTitle = $params[0];
+	$route['title'] .= ' - ' . strtoupper($newTitle[0]) . strtolower(substr($newTitle, 1));
+}
+# Select page to display
+if (!empty($params) && $params[0] == 'view') {
+	$view->display('/view-survey-results.php');
+} else if (!empty($params) && $params[0] == 'results') {
+    $view->display('surveys/results.php');
+} else {
+	include('header.php');
+	global $db;
+	$filter = "";
+	
+	if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+		$filter = isset($_POST['filter']) ? "%" . $_POST['filter'] . "%" : "%%";
+		$result = $db->query("SELECT * FROM curricula WHERE LOWER(curriculumname::text) LIKE LOWER($1) " .
+			"OR LOWER(curriculumtype::text) LIKE LOWER($1) ORDER BY curriculumname", [$filter]);
+	} else {
+		//$result = $db->query("SELECT * FROM curricula ORDER BY curriculumname", []);
+	}
+	?>
+	<div style="width: 100%; height: 100%">
+		
+		
+		<!------------------------ This is the update button ------------------------->
+		
+		<?php
+			function updateDB(){	
+				//$command = escapeshellcmd('py C:\Program Files (x86)\EasyPHP-Devserver-17\eds-www\views\surveys\pullAndParseSurveys.py');
+				$output = shell_exec('py "C:\Program Files (x86)\EasyPHP-Devserver-17\eds-www\views\surveys\pullAndParseSurveys.py"');
```

Comment:
Probably shouldn't use an absolute path since this is not how it will be configured on the server. If you need to do an absolute path, add configuration to the `config.ini` file that specifies this file path.